Fix JSDoc param mismatch blocking lint-required

## Problem

\`scripts/global/governance-audit-coverage.js:105\` has a JSDoc \`@param\` named \`argv\` but the function signature uses \`_argv\` (underscore prefix indicating unused). This triggers \`jsdoc/check-param-names\` as a hard error, failing the \`lint-required\` CI gate on every PR that runs against current main.

## Evidence

\`\`\`
scripts/global/governance-audit-coverage.js
  105:1   error    Expected @param names to be "_argv". Got "argv"  jsdoc/check-param-names
✖ 1400 problems (1 error, 1399 warnings)
\`\`\`

This is the **single error** in the entire repo lint run; the remaining 1399 issues are warnings (mostly \`jsdoc/require-jsdoc\` in \`scripts/wiki/*.js\`). Fixing this one line unblocks \`lint-required\` for all open PRs.

## Acceptance criteria

- AC1: \`scripts/global/governance-audit-coverage.js\` JSDoc \`@param\` is updated from \`argv\` to \`_argv\` to match the function signature.
- AC2: \`npm run lint\` exits with \`0 errors\` (warnings tolerated).
- AC3: \`lint-required\` CI gate passes on the linked PR.

## Lane + test_strategy

- lane: lane:code-change
- test_strategy: tdd-pyramid (covered by existing lint-js script invocation; no new test file needed since the lint check IS the test)

## Note

Surfaced while shipping #2113. The IDE auto-suggested this exact fix; discarded at the time because it was outside #2113 scope. Filing as the proper isolated ticket.

--- a/.changes/unreleased/2113.md
+++ b/.changes/unreleased/2113.md
@@ -1,0 +1,8 @@
+### Changed
+
+- `.gitignore` extended with root-anchored `/dist/` + `/planning/` patterns (matching `/tickets/` from #2051). Build output + operator-local planning area now never tracked; cleans up the cross-team residue accumulation pattern that triggered false-positive stop-hook alarms (~20 alarms per session observed 2026-05-21/22). (#2113)
+- `scripts/analyze-governance-drift.js` adopted into the tracked set (was an orphaned residue file from a prior session; now a legitimate governance utility). (#2113)
+
+### Notes
+
+- Structural prevention: Epic #2091 Phase-1 children (#2103 session-start state rotation, #2106 per-session state keying, #2107 canonical-main read-only enforcer — shipped) eliminate future residue accumulation. This ticket is a one-time adoption of pre-existing residue.

--- a/.gitignore
+++ b/.gitignore
@@ -53,5 +53,10 @@ logs/copilot-usage.json
 /tickets/
 .worktrees/
 
+# Build output + operator-local planning area (#2113): never committed.
+# Root-anchored so wiki/work-log/planning/ (#2054, Wiki B) can exist if filed.
+/dist/
+/planning/
+
 # Operator-local spike outputs (Wave 1 #891/#892/#893) — never committed
 tmp/

--- a/scripts/analyze-governance-drift.js
+++ b/scripts/analyze-governance-drift.js
@@ -1,10 +1,14 @@
 'use strict';
 const cp = require('child_process');
 
-const raw = cp.execFileSync('gh', ['issue', 'list', '--state', 'open', '--limit', '500', '--json', 'number,title,body,labels'], { encoding: 'utf8', maxBuffer: 50 * 1024 * 1024 });
+const OPEN_LIMIT = '500';
+const ALL_LIMIT = '1000';
+const MAX_BUFFER_BYTES = 50 * 1024 * 1024;
+
+const raw = cp.execFileSync('gh', ['issue', 'list', '--state', 'open', '--limit', OPEN_LIMIT, '--json', 'number,title,body,labels'], { encoding: 'utf8', maxBuffer: MAX_BUFFER_BYTES });
 const issues = JSON.parse(raw);
 
-const rawAllEpics = cp.execFileSync('gh', ['issue', 'list', '--limit', '1000', '--json', 'number,title,body,labels'], { encoding: 'utf8', maxBuffer: 50 * 1024 * 1024 });
+const rawAllEpics = cp.execFileSync('gh', ['issue', 'list', '--limit', ALL_LIMIT, '--json', 'number,title,body,labels'], { encoding: 'utf8', maxBuffer: MAX_BUFFER_BYTES });
 const allIssues = JSON.parse(rawAllEpics);
 const epicNumbers = new Set(allIssues.filter(i => i.labels.some(l => l.name === 'type:epic')).map(i => i.number));
 

--- a/scripts/analyze-governance-drift.js
+++ b/scripts/analyze-governance-drift.js
@@ -1,0 +1,66 @@
+'use strict';
+const cp = require('child_process');
+
+const raw = cp.execFileSync('gh', ['issue', 'list', '--state', 'open', '--limit', '500', '--json', 'number,title,body,labels'], { encoding: 'utf8', maxBuffer: 50 * 1024 * 1024 });
+const issues = JSON.parse(raw);
+
+const rawAllEpics = cp.execFileSync('gh', ['issue', 'list', '--limit', '1000', '--json', 'number,title,body,labels'], { encoding: 'utf8', maxBuffer: 50 * 1024 * 1024 });
+const allIssues = JSON.parse(rawAllEpics);
+const epicNumbers = new Set(allIssues.filter(i => i.labels.some(l => l.name === 'type:epic')).map(i => i.number));
+
+const childTickets = [];
+const trueIndependent = [];
+const activeEpics = [];
+const driftFindings = [];
+
+for (const issue of issues) {
+  const isEpic = issue.labels.some(l => l.name === 'type:epic');
+  if (isEpic) {
+    const isDone = issue.labels.some(l => l.name === 'status:done');
+    if (!isDone) activeEpics.push(issue);
+    continue;
+  }
+  
+  const body = issue.body || '';
+  const title = issue.title || '';
+  const parentMatch = body.match(/parent\s*(?:epic)?\s*:\s*#?(\d+)/i) || body.match(/part\s*of\s*#?(\d+)/i);
+  let parentEpicNum = parentMatch ? Number(parentMatch[1]) : null;
+  
+  const titleEpicMatch = title.match(/\(Epic\s*#?(\d+)\)/i);
+  if (titleEpicMatch && !parentEpicNum) {
+    parentEpicNum = Number(titleEpicMatch[1]);
+    driftFindings.push({
+      issue: issue.number,
+      title: issue.title,
+      type: 'Implicit Parent Drift',
+      message: `Mentions parent Epic #${parentEpicNum} in title, but lacks standard "Parent Epic: #${parentEpicNum}" metadata in its body.`
+    });
+  }
+  
+  if (!parentEpicNum) {
+    for (const epicNum of epicNumbers) {
+      const regex = new RegExp(`\\b(?:Epic|Issue|Parent|under|for)\\s+#${epicNum}\\b`, 'i');
+      if (regex.test(body) || regex.test(title)) {
+        parentEpicNum = epicNum;
+        driftFindings.push({
+          issue: issue.number,
+          title: issue.title,
+          type: 'Unstructured Epic Association',
+          message: `References Epic #${epicNum} without formal parent-child metadata declaration.`
+        });
+        break;
+      }
+    }
+  }
+
+  const isDone = issue.labels.some(l => l.name === 'status:done');
+  if (!isDone) {
+    if (parentEpicNum) {
+      childTickets.push({ number: issue.number, title: issue.title, parentEpicNum });
+    } else {
+      trueIndependent.push(issue);
+    }
+  }
+}
+
+console.log(JSON.stringify({ activeEpics, trueIndependent, childTickets, driftFindings }, null, 2));

--- a/scripts/global/governance-audit-coverage.js
+++ b/scripts/global/governance-audit-coverage.js
@@ -102,7 +102,7 @@ function writeReport(report) {
 }
 
 /** CLI entry: read catalog, audit, write, set exit code.
- * @param {string[]} argv - process argv slice.
+ * @param {string[]} _argv - process argv slice (unused).
  * @returns {number} exit code (0 ok, 1 violations, 2 degraded parse failure). */
 function main(_argv) {
   let md;

